test_app/tests/authentication/serializers/test_authententicator_map.py

```diff
@@ -0,0 +1,153 @@
+from unittest.mock import MagicMock
+
+import pytest
+from rest_framework.serializers import ValidationError
+
+from ansible_base.authentication.serializers.authenticator_map import AuthenticatorMapSerializer
+from ansible_base.lib.utils.auth import get_organization_model, get_team_model
+from ansible_base.rbac import permission_registry
+from ansible_base.rbac.models import RoleDefinition
+
+SYSTEM_ROLE_NAME = 'System role'
+TEAM_ROLE_NAME = 'Team Member'
+ORG_ROLE_NAME = 'Organization Member'
+
+
+@pytest.fixture
+def serializer():
+    return AuthenticatorMapSerializer()
+
+
+@pytest.fixture
+def system_role():
+    return RoleDefinition.objects.create(
+        name=SYSTEM_ROLE_NAME,
+    )
+
+
+@pytest.fixture
+def team_role():
+    return RoleDefinition.objects.create(
+        name=TEAM_ROLE_NAME,
+        content_type=permission_registry.content_type_model.objects.get_for_model(get_team_model()),
+        managed=True,
+    )
+
+
+@pytest.fixture
+def organization_role():
+    return RoleDefinition.objects.create(
+        name=ORG_ROLE_NAME,
+        content_type=permission_registry.content_type_model.objects.get_for_model(get_organization_model()),
+        managed=True,
+    )
```

Comment:
These seem like they might be useful in a larger context should we move them either into `conftest` or into `lib/testing/fixtures.pm`?

Reply:
in case of larger context, fixtures and constants must be renamed, because they're creating member roles. 

I can move it to conftest, but rather not to lib's fixtures - rbac is creating it with permissions etc., which is not needed for these tests, but can be required somewhere else